docs/C-Data-Structures/arrays.md

```diff
@@ -86,13 +100,17 @@ This coding style facilitates modifiability.  If we change the size, we need to
 
 ## Elements
 
-Each element has a unique index and holds a single value.  Index numbering starts at 0 and extends to one less than the the number of elements in the array.  To refer to a specific element, we write the array name followed by bracket notation around the element's index.
+Each element has a unique index and holds a single value.  Index numbering starts at 0 and extends to one less than the number of elements in the array.  To refer to a specific element, we write the array name followed by bracket notation around the element's index.
 
-**`identifier[index]`**
+```c
+identifier[index]
+```
 
 For example, to access the first element of grade, we write:
 
-**`grade[0]`**
+```c
+grade[0]
+```
 
 To display all elements of grade, we iterate:
```

Comment:
You should add "int i = 0"

Reply:
Sorry what do you mean? Like grade[int i = 0]? I don't think you need to initialize the index before you use it.

Reply:
I mean on line 119, it should be 
```
for (int i = 0; i < NGRADES; i++) {
```

Reply:
Oh got it. Nice catch! Thanks :)